Sources/AsyncHTTPClient/Utils.swift

```diff
@@ -230,3 +289,17 @@ extension Connection {
         }.recover { _ in }
     }
 }
+
+extension NIOClientTCPBootstrap {
+    var isNIOTS: Bool {
+        #if canImport(Network)
+            if #available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *) {
+                return self.underlyingBootstrap is NIOTSConnectionBootstrap
+            } else {
+                return false
+            }
+        #else
+            return false
+        #endif
+    }
```

Comment:
Is there a reason, why we don't use this pattern?
```suggestion
    var isNIOTS: Bool {
        #if canImport(Network)
            if #available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *) {
                return self.underlyingBootstrap is NIOTSConnectionBootstrap
            }
        #endif
        return false
    }
```

Reply:
I reduced the amount of code we need here but happy to add another `} else { return false }`